### ISSUE TITLE
Colab notebook demo IndexError

### ISSUE THREAD

**Issue by nbroad1881** [kingoflolz/mesh-transformer-jax#49]
This doesn't run correctly for me anymore. A day or two ago it was working fine.

https://colab.research.google.com/github/kingoflolz/mesh-transformer-jax/blob/master/colab_demo.ipynb

```python
total_batch = per_replica_batch * jax.device_count() // cores_per_replica

network = CausalTransformer(params)

network.state = read_ckpt(network.state, "step_383500/", devices.shape[1])

network.state = network.move_xmap(network.state, np.zeros(cores_per_replica))
```
```
FilteredStackTrace: IndexError: Too many indices for array: 4 non-None/Ellipsis indices for dim 3.

The stack trace above excludes JAX-internal frames.
The following is the original exception that occurred, unmodified.

--------------------

The above exception was the direct cause of the following exception:

IndexError                                Traceback (most recent call last)
/usr/local/lib/python3.7/dist-packages/jax/_src/numpy/lax_numpy.py in _canonicalize_tuple_index(arr_ndim, idx)
   4860   if len_without_none > arr_ndim:
   4861     msg = "Too many indices for array: {} non-None/Ellipsis indices for dim {}."
-> 4862     raise IndexError(msg.format(len_without_none, arr_ndim))
   4863   ellipses = (i for i, elt in enumerate(idx) if elt is Ellipsis)
   4864   ellipsis_index = next(ellipses, None)

IndexError: Too many indices for array: 4 non-None/Ellipsis indices for dim 3.
```

**Comment by jmerizia** [comment]
Until it's fixed, you can revert to `06d6ae27f7247a82d135590b1bd1cd3a5998862`. So add this after git clone:

```
!cd mesh-transformer-jax && git checkout 06d6ae27f7247a82d135590b1bd1cd3a59988624
```

**Comment by kingoflolz** [comment]
Fixed in 831f405, sorry about that
